cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -85,3 +85,315 @@ Feature: Accessibility tests - Common list
       | Decimal-Input    |
       | GroupedCharacter |
       | Number-Input     |
+
+  @accessibility
+  Scenario Outline: Accordion component <story> page closed state
+    When I open "Accordion" component page "<story>"
+    Then "Accordion <story> page" component has no accessibility violations
+    Examples:
+      | story                |
+      | default story        |
+      | with dynamic content |
+
+  @accessibility
+  Scenario: Accordion component primary page opened state
+    Given I open "Accordion" component page "default story"
+    When I expand default accordionRow via click
+    Then "Accordion default page" component has no accessibility violations
+
+  @accessibility
+  Scenario Outline: <component> default story page
+    When I open "Test <component>" component page "default"
+    Then "<component>" component has no accessibility violations
+    Examples:
```

Comment:
Why is there an Examples table with just one example? Is the intention to expand this to include more examples to warrant having a table?